samples/proto/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+#![allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
is this necessary? I don't see anything that would require it. is it related to code generated by the macros?

Reply:
Removed.